docs/cody/usage-and-pricing.mdx

```diff
@@ -82,11 +82,11 @@ If you change your mind after canceling your plan please contact our Support tea
 
 ### Are there any refunds for the Pro subscription?
 
-We don't offer refunds, but if you have any queries regarding the Cody Pro plan, please write to support@sourcegraph.com, and we'll help resolve the issue.
+We don't offer refunds, but if you have any queries regarding the Cody Pro plan, please write to support@sourcegraph.com or visit our [support forum](https://community.sourcegraph.com/), and we'll help resolve the issue.
```

Comment:
@Kynlos, for refund/billing issues, we want them to email since there are private details.

Reply:
Ah ok, maybe a change to "if you have any queries regarding billing for the Cody Pro Plan"?

Reply:
We will just leave this as be :)